sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/manager/FeatureManager.java

```diff
@@ -57,9 +58,22 @@ public class FeatureManager {
      * @return state of the feature
      * @throws FilterNotFoundException file not found
      */
-    public Mono<Boolean> isEnabledAsync(String feature) throws FilterNotFoundException {
+    public Mono<Boolean> isEnabledAsync(String feature) {
         return Mono.just(checkFeatures(feature));
     }
+    
+    /**
+     * Checks to see if the feature is enabled. If enabled it check each filter, once a single filter returns true it
+     * returns true. If no filter returns true, it returns false. If there are no filters, it returns true. If feature
+     * isn't found it returns false.
+     *
+     * @param feature Feature being checked.
+     * @return state of the feature
+     * @throws FilterNotFoundException file not found
+     */
+    public Boolean isEnabled(String feature) throws FilterNotFoundException {
+        return checkFeatures(feature);
+    }
 
     private boolean checkFeatures(String feature) throws FilterNotFoundException {
```

Comment:
How about rename this method to make it easier to understand. For example: `featureIsOnAndPassedFilter`

Reply:
I dislike giving it a longer name as doesn't tell the whole story. Would rather make it a JavaDoc. Changing it to `checkFeature` though as it only checks 1 at a time.